includes/definitions/lastmile.yaml

```diff
@@ -0,0 +1,11 @@
+os: lastmile
```

Comment:
This looks like it's the vendors name. The OS name needs to be what the firmware is called or if that's not possible the device / type of device.